clients/ptranslator/src/main/java/org/hawkular/metrics/clients/ptrans/ConfigurationKey.java

```diff
@@ -1,87 +0,0 @@
-/*
- * Copyright 2014-2016 Red Hat, Inc. and/or its affiliates
- * and other contributors as indicated by the @author tags.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.hawkular.metrics.clients.ptrans;
-
-/**
- * PTrans configuration keys.
- *
- * @author Thomas Segismont
- */
-public enum ConfigurationKey {
```

Comment:
Do you mind updating/reverting all the code that now has strings literals but were originating from this class?